composer/workflows/gke_operator.py

```diff
@@ -0,0 +1,244 @@
+        # The var template variable allows you to access variables defined in
+        # Airflow UI. In this case we are getting the value of my_value and
+        # setting the environment variable `MY_VALUE`. The pod will fail if
+        # `my_value` is not set in the Airflow UI.
+        env_vars={"MY_VALUE": "{{ var.value.my_value }}"},
+    )
+    # [END composer_gkeoperator_templateconfig]
+
+    # [START composer_gkeoperator_affinity]
+    kubernetes_affinity_ex = GKEStartPodOperator(
+        task_id="ex-pod-affinity",
+        project_id=PROJECT_ID,
+        location=CLUSTER_ZONE,
+        cluster_name=CLUSTER_NAME,
+        name="ex-pod-affinity",
+        namespace="default",
+        image="perl",
+        cmds=["perl"],
+        arguments=["-Mbignum=bpi", "-wle", "print bpi(2000)"],
+        # affinity allows you to constrain which nodes your pod is eligible to
+        # be scheduled on, based on labels on the node. In this case, if the
+        # label 'cloud.google.com/gke-nodepool' with value
+        # 'nodepool-label-value' or 'nodepool-label-value2' is not found on any
+        # nodes, it will fail to schedule.
+        affinity={
+            "nodeAffinity": {
+                # requiredDuringSchedulingIgnoredDuringExecution means in order
+                # for a pod to be scheduled on a node, the node must have the
+                # specified labels. However, if labels on a node change at
+                # runtime such that the affinity rules on a pod are no longer
+                # met, the pod will still continue to run on the node.
+                "requiredDuringSchedulingIgnoredDuringExecution": {
+                    "nodeSelectorTerms": [
+                        {
+                            "matchExpressions": [
+                                {
+                                    # When nodepools are created in Google Kubernetes
+                                    # Engine, the nodes inside of that nodepool are
+                                    # automatically assigned the label
+                                    # 'cloud.google.com/gke-nodepool' with the value of
+                                    # the nodepool's name.
+                                    "key": "cloud.google.com/gke-nodepool",
+                                    "operator": "In",
+                                    # The label key's value that pods can be scheduled
+                                    # on.
+                                    "values": [
+                                        "pool-1",
+                                    ],
+                                }
+                            ]
+                        }
+                    ]
+                }
+            }
+        },
+    )
+    # [END composer_gkeoperator_affinity]
+    # [START composer_gkeoperator_fullconfig]
+    kubernetes_full_pod = GKEStartPodOperator(
+        task_id="ex-all-configs",
+        name="pi",
```

Comment:
This is just me not at expert in GKE, but out of curiosity, what does 'pi' mean here?

Reply:
```suggestion
        name="full",
```